src/atopile/cli/build.py

```diff
@@ -50,10 +52,34 @@ def build(
     build_ctxs = create_build_contexts(entry, build, target, option, standalone)
 
     for build_ctx in build_ctxs:
+        if keep_picked_parts is not None:
+            build_ctx.keep_picked_parts = keep_picked_parts
+
+        if keep_net_names is not None:
+            build_ctx.keep_net_names = keep_net_names
+
         if frozen is not None:
             build_ctx.frozen = frozen
             if frozen:
-                build_ctx.keep_picked_parts = True
+                if keep_picked_parts is False:  # is, ignores None
+                    with downgrade(ValueError):
```

Comment:
Tempted to make these errors on conflict, rather than warning on False, but I can see some utility - mainly for dev